### ISSUE TITLE
AttributeError when processing replay file

### ISSUE THREAD

**Issue by Jondeboy** [gabriel-oana/wotreplay#6]
Im getting the following error when processing the replay file attached: 
 
 File "C:\Python38\lib\site-packages\wotreplay\wotreplay.py", line 12, in __init__
    self.replay = Replay(file=file_path, db_path=db_path, db_name=db_name, load=load)
  File "C:\Python38\lib\site-packages\wotreplay\action\extract_data_from_replay.py", line 51, in __init__
    self._assign_replay_date()
  File "C:\Python38\lib\site-packages\wotreplay\action\extract_data_from_replay.py", line 83, in _assign_replay_date
    self.replay_date = Extractor.get_replay_date(self.meta_data)
  File "C:\Python38\lib\site-packages\wotreplay\helper\extractor.py", line 23, in get_replay_date
    replay_date = metadata.get('dateTime')
AttributeError: 'NoneType' object has no attribute 'get'

[20200812_2222_usa-A05_M4_Sherman_115_sweden.zip](https://github.com/gabriel-oana/wotreplay/files/10047916/20200812_2222_usa-A05_M4_Sherman_115_sweden.zip)

**Comment by Jondeboy** [comment]
I was looping through my replays (over 9000) and this one didn't go through. It is possible that this file is corrupted.

**Comment by gabriel-oana** [comment]
I have noticed some issues like this when I used to play the game and parsing various replays. 
Some replays (generally speaking a small number), are not recorded properly by the game client and is very little we can do about it. 

Best to do if you process large files is something like this.
https://github.com/gabriel-oana/wotreplay/blob/main/wotreplay/wotreplay.py#L52

Notice that I use a "try" method which doesn't raise an issue when one file is broken
